src/App.tsx

```diff
@@ -9,45 +9,82 @@ import { TrailerFilm } from "./Components/molecules/trailerFilm";
 import { films, trailer } from "./mock";
 import { useState } from "react";
 import { Button } from "./Components/atoms/button/Button";
-import { filterBy, sortBy } from "./helper/helper";
-import { FilterFilm } from "./Components/atoms/filterFilmBtn/FilterFilm";
+import { useEffect } from "react";
+import {
+  filterBy,
+  filterByGenre,
+  getFilterProps,
+  sortBy,
+} from "./helper/helper";
+import { ButtonBookMark } from "./Components/atoms/buttonBookmark";
+import { Switch } from "./Components/atoms/switch";
```

Comment:
please combine the imports
import { Switch, ButtonBookMark  } from "./Components/atoms";